ansible/roles/debops-contrib.etckeeper/tasks/main.yml

```diff
@@ -111,6 +111,14 @@
   tags: [ 'role::etckeeper:vcs_config' ]
 ## ]]]
 
+- name: Unstage ignored files from Git index
+  shell: etckeeper vcs ls-files -i --exclude-standard -z | xargs -0 etckeeper vcs reset --mixed --
+  register: git_reset
```

Comment:
You should namespace this variable, something like: `etckeeper__register_git_reset`.